lib/clickhousex/protocol.ex

```diff
@@ -6,11 +6,10 @@ defmodule Clickhousex.Protocol do
   alias Clickhousex.Error
   alias Clickhousex.HTTPClient, as: Client
 
-  defstruct conn_opts: [], base_address: "", conn: nil
+  defstruct conn_opts: [], conn: nil
 
   @type state :: %__MODULE__{
           conn_opts: Keyword.t(),
-          base_address: String.t(),
```

Comment:
`base_address` wasn't set in `connect` so it's unlikely it was used anywhere at all